docs/Settings.md

```diff
@@ -166,8 +166,8 @@ These parameters are accessed by calling [GetParam](./api/GetParam.md) or [SetPa
 | `QUIC_PARAM_CONN_TLS_SECRETS`<br> 19              | QUIC_TLS_SECRETS              | Set-only  | The TLS secrets struct to be populated by MsQuic.                                         |
 | `QUIC_PARAM_CONN_VERSION_SETTINGS`<br> 20         | QUIC_VERSION_SETTINGS         | Both      | The desired QUIC versions for the connection.                                             |
 | `QUIC_PARAM_CONN_CIBIR_ID`<br> 21                 | uint8_t[]                     | Set-only  | The CIBIR well-known identifier.                                                          |
-| `QUIC_PARAM_CONN_STATISTICS_V2`<br> 5             | QUIC_STATISTICS_V2            | Get-only  | Connection-level statistics, version 2.                                                   |
-| `QUIC_PARAM_CONN_STATISTICS_V2_PLAT`<br> 6        | QUIC_STATISTICS_V2            | Get-only  | Connection-level statistics with platform-specific time format, version 2.                |
+| `QUIC_PARAM_CONN_STATISTICS_V2`<br> 16            | QUIC_STATISTICS_V2            | Get-only  | Connection-level statistics, version 2.                                                   |
```

Comment:
The values should be 22 and 23 (0x16 and 0x17).
```
#define QUIC_PARAM_CONN_STATISTICS_V2                   0x05000016  // QUIC_STATISTICS_V2
#define QUIC_PARAM_CONN_STATISTICS_V2_PLAT              0x05000017  // QUIC_STATISTICS_V2
```

IMO, keeping track of the values in this table is error prone anyways. We should only have only one source of truth.

Reply:
I'd be fine with just removing the numeric values altogether.

Reply:
Ahh of course, hexadecimal. I got a little confused because of the 5 in 0x05... so I thought the docs used the last XX digits (kept in base 16), but I can push another update!